### PR TITLE
fix(licenses): Fixed a broken link

### LINKED ISSUE #6907: Issue: New Relic Agent Software Notice
## How can we make our docs better?

This link is broken and does not send me anywhere
>> https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile/get-started 

### Doc information (don't delete this section)

* https://docs.newrelic.com/docs/licenses/license-information/distributed-licenses/new-relic-agent-software-notice/
* **OS**: Mac OS 10.15.7
* **Browser**: Chrome 100.0.4896.60
* **Device**: Unknown

### PR DESCRIPTION
**Pull request by paperclypse**
Closes #6907. The old link was pointing to a category level.

### CODE REVIEW

**Comment by paperclypse** [comment]
This probably falls in the "doesn't really need a review" category :D

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
Figured it would be easier to give you a screenshot than try to explain what I mean.

File: src/content/docs/licenses/license-information/distributed-licenses/new-relic-agent-software-notice.mdx [hunk 1]
```diff
@@ -29,6 +29,6 @@ The Subscription Agreement will control in event of a conflict with this notice.
...
 
-For avoidance of doubt, you may copy and distribute New Relic agents pursuant to your Subscription Agreement for New Relic Browser and New Relic Mobile and as set forth: [https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile/get-started](/docs/mobile-monitoring/new-relic-mobile/get-started) and [https://docs.newrelic.com/docs/browser/new-relic-browser/installation/install-new-relic-browser-agent](/docs/browser/new-relic-browser/installation/install-new-relic-browser-agent)
+For avoidance of doubt, you may copy and distribute New Relic agents pursuant to your Subscription Agreement for New Relic Browser and New Relic Mobile and as set forth: [/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/](/docs/mobile-monitoring/new-relic-mobile/get-started) and [https://docs.newrelic.com/docs/browser/new-relic-browser/installation/install-new-relic-browser-agent](/docs/browser/new-relic-browser/installation/install-new-relic-browser-agent)
```
Comment (homelessbirds) [inline src/content/docs/licenses/license-information/distributed-licenses/new-relic-agent-software-notice.mdx:32]:
```suggestion
For avoidance of doubt, you may copy and distribute New Relic agents pursuant to your Subscription Agreement for New Relic Browser and New Relic Mobile and as set forth: [/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/](/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/) and [https://docs.newrelic.com/docs/browser/new-relic-browser/installation/install-new-relic-browser-agent](/docs/browser/new-relic-browser/installation/install-new-relic-browser-agent)
```

**Review by homelessbirds — APPROVED** [review]
I fixed the link :)

**Comment by paperclypse** [comment]
@barbnewrelic I think it's meant to be that way, because it's a licensing doc.

**Pull request merged**